user-guide/enterprise-auth/intro.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Using Enterprise Authentication
+description: Learn how to connect to your enterprise authentication provider, and how to leverage data from that provider in your apps.
+position: 1
+publish: true
+slug: enterprise-auth
```

Comment:
I have added slug to each document for easier navigation. They also work great if later we need to change the docs structure.